llama_stack/apis/tools/rag_tool.py

```diff
@@ -86,6 +88,8 @@ class RAGQueryConfig(BaseModel):
     max_chunks: int = 5
     chunk_template: str = "Result {index}\nContent: {chunk.content}\nMetadata: {metadata}\n"
     mode: str | None = None
+    reranker_type: str = "rrf"
```

Comment:
we should make these optional, it's confusing if a user chooses vector or keyword and has to set the reranker type, no?